includes/class-wc-install.php

```diff
@@ -628,6 +626,24 @@ private static function get_schema() {
   PRIMARY KEY (log_id),
   KEY level (level)
 ) $collate;
+CREATE TABLE {$wpdb->prefix}woocommerce_webhooks (
+  webhook_id BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+  status varchar(200) NOT NULL,
+  name text NOT NULL,
```

Comment:
Because no need to limit lengths, is just for identification. It's a free field.